nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -1079,6 +1079,19 @@ class ChatRequest(BaseModel):
         title="Filters",
         description="The list of filters to apply. Filtering examples can be found here: https://docs.nuclia.dev/docs/rag/advanced/search/#filters",  # noqa: E501
     )
+    keyword_filters: Union[list[str], list[Filter]] = Field(
+        default=[],
+        title="Keyword filters",
+        description=(
+            "List of keyword filter expressions to apply to the retrieval step. "
+            "The text block search will only be performed on the documents that contain the specified keywords. "
```

Comment:
Could you make more clear than keyword can be a sentence? Add a longer example or a comment on it. It feels that each term should be a single word

Reply:
Ok! I will add an example with a name + surname